target/linux/generic/hack-6.12/931-fix-compilation-warnings.patch

```diff
@@ -0,0 +1,23 @@
+--- a/include/linux/irq.h
++++ b/include/linux/irq.h
+@@ -1267,6 +1267,9 @@ static inline u32 irq_reg_readl(struct i
+ 		return readl(gc->reg_base + reg_offset);
+ }
+ 
++int get_c0_perfcount_int(void);
++unsigned int get_c0_compare_int(void);
++
+ struct irq_matrix;
+ struct irq_matrix *irq_alloc_matrix(unsigned int matrix_bits,
+ 				    unsigned int alloc_start,
+--- a/arch/mips/include/asm/switch_to.h
++++ b/arch/mips/include/asm/switch_to.h
+@@ -111,7 +111,7 @@ do {									\
+ 	__mips_mt_fpaff_switch_to(prev);				\
+ 	lose_fpu_inatomic(1, prev);					\
+ 	if (tsk_used_math(next))					\
+-		__sanitize_fcr31(next);					\
++		{ __sanitize_fcr31(next); }				\
```

Comment:
This has already been [fixed upstream since 6.12.2](https://github.com/gregkh/linux/commit/2a375beb4c3824bb984e65860ad7ec0197be91f7)

Reply:
The history of this PR goes back to September 2024 and kernel 6.12, thanks for the info, I didn't notice it was patched in the meantime.